<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>30</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>30</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>30</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit{
	border: none;
	border-bottom: 1px solid;
	border-bottom-color: rgb(235, 235, 235);
	background-color: rgb(250, 250, 250);
	font: 10pt url(:/plugins/StaraMaps/Roboto-Regular.ttf);
}

QLineEdit::hover{
	border-bottom: 2px solid;
	border-bottom-color: rgb(225, 225, 225);
}

QLineEdit:focus{
	border-bottom-color: rgb(243, 116, 53);
}

QLineEdit:disabled{
	background-color: rgb(230, 230, 230);
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 170, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Value1</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>----</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignLeft">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Value2</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="newLineEdit">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
